.changeset/chilly-wasps-fold.md

```diff
@@ -0,0 +1,5 @@
+---
+'@itwin/itwinui-react': minor
+---
+
+`LabeledInput` & `LabeledTextarea` now internally use `iui-input-with-icon` instead of `InputWithDecorations`.
```

Comment:
These are implementation details that should not be part of react changeset. Try to think about how it affects our users.

```suggestion
`LabeledInput` and `LabeledTextarea` have been slightly reworked to fix alignment and consistency issues when used with `svgIcon`.
```